auto-diagram/headless_chrome/screenshot2.js

```diff
@@ -0,0 +1,62 @@
+const CDP = require('chrome-remote-interface');
```

Comment:
Is this a "backup" file that can/should be removed?

Reply:
Cool - that makes sense. In the future, let's talk about how to use environment variables to simplify this code.